nuxeo-features/nuxeo-platform-csv-export/src/test/java/org/nuxeo/ecm/platform/csv/export/TestCSVExportAction.java

```diff
@@ -87,10 +88,12 @@
 import com.google.common.hash.Hashing;
 
 @RunWith(FeaturesRunner.class)
-@Features({ CoreBulkFeature.class, CoreFeature.class, DirectoryFeature.class })
+@Features({ CoreBulkFeature.class, CoreFeature.class, DirectoryFeature.class,
+        RepositoryLightElasticSearchFeature.class })
 @Deploy("org.nuxeo.ecm.default.config")
 @Deploy("org.nuxeo.ecm.platform.csv.export")
 @Deploy("org.nuxeo.ecm.core.test.tests:OSGI-INF/test-repo-core-types-contrib.xml")
+@Deploy("org.nuxeo.elasticsearch.core.test:elasticsearch-test-contrib.xml")
```

Comment:
Isn't this contribution already deployed by the RepositoryLightElasticSearchFeature`?